ibis/expr/api.py

```diff
@@ -1783,7 +1800,8 @@ def _timestamp_strftime(arg, format_str):
     minute=_extract_field('minute', _ops.ExtractMinute),
     second=_extract_field('second', _ops.ExtractSecond),
     millisecond=_extract_field('millisecond', _ops.ExtractMillisecond),
-    truncate=_timestamp_truncate
+    truncate=_timestamp_truncate,
+    time=_timestamp_time,
```

Comment:
this is very different though. ``.time()`` simply gives back a time node w/o actually doing any ting under the hood. Then you have to calll ``.between(...)`` to actually do anything